operator/gitops/argocd/pipeline-service/openshift-pipelines/allow-argocd-to-manage.yaml

```diff
@@ -14,6 +14,16 @@ rules:
       - patch
       - create
       - delete
+  - apiGroups:
+      - security.openshift.io
+    resources:
+      - securitycontextconstraints
+    verbs:
+      - get
+      - list
+      - patch
+      - create
+      - delete
```

Comment:
do we really need delete ?   In the wrong hands could be destructive

Reply:
I'm guessing that perhaps the openshift pipelines operator had this as part of doing uninstall?

If so, I guess ok .... but if somebody ever hacked a pod with this permission ....

Reply:
Those permissions are needed to create the pipelines-scc resource. It is a temporary solution and will be removed in the future.

Reply:
ok